Lib/fontbakery/commands/check_specification.py

```diff
@@ -268,6 +274,78 @@ def main(specification=None, values=None):
     import json
     json.dump(sr.getdoc(), args.json, sort_keys=True, indent=4)
 
+  if args.ghmarkdown:
```

Comment:
I'm disappointed that you don't implement this as a reporter. It's clearly the task of a reporter to do this job and there are examples for it.
Maybe you can import and use the SerializeReporter within a MarkdownReporter, in order not to rewrite your code too much, but having the formatting routines in `check_specification.py` is just wrong.